modules/administration_manual/pages/configuration/user/user_provisioning_api.adoc

```diff
@@ -52,7 +48,6 @@ sending a basic HTTP authentication header.
 * 103 - unknown error occurred whilst adding the user
 * 104 - group does not exist
 
-[[example]]
```

Comment:
there is a couple of times the header "example" (and xml-output) present.
without manual ID, any identical header becomes its own ID which then is a dupe.
how will that work ?

Reply:
I'll need to update the headers, so that that doesn't happen. I'm guessing that the auto-generated id process wouldn't create duplicates. However, I'm curious to be sure, and if it doesn't, to find out how names are created.

Reply:
I've just had a look at this and any subsequent headers are auto-generated as example-1, example-2, etc. I've gone through the files and added the section header as a prefix, such as "Add User Example".